x-pack/plugins/infra/public/pages/logs/settings/indices_configuration_panel.tsx

```diff
@@ -25,35 +27,100 @@ export const IndicesConfigurationPanel = React.memo<{
+              <FormattedMessage
+                id="xpack.infra.logSourceConfiguration.logSourcesTitle"
+                defaultMessage="Log sources"
+              />
+            </h3>
+          </EuiTitle>
+        ),
+      }}
+    >
+      <EuiCheckableCard
+        id="dataView"
+        label={
+          <EuiTitle size="xs">
+            <h2>
+              <FormattedMessage
+                id="xpack.infra.logSourceConfiguration.dataViewSectionTitle"
+                defaultMessage="Data view (recommended)"
+              />
+            </h2>
+          </EuiTitle>
+        }
+        name="dataView"
+        value="dataView"
+        checked={isIndexPatternFormElement(indicesFormElement)}
+        onChange={() => {
+          if (indicesFormElement.initialValue?.type === 'index_pattern') {
+            indicesFormElement.updateValue(() => indicesFormElement.initialValue);
+          } else {
+            indicesFormElement.updateValue(() => ({
+              type: 'index_pattern',
+              indexPatternId: '',
+            }));
+          }
 
-  if (isIndexPatternFormElement(indicesFormElement)) {
-    return (
-      <IndexPatternConfigurationPanel
-        isLoading={isLoading}
-        isReadOnly={isReadOnly}
-        indexPatternFormElement={indicesFormElement}
-      />
-    );
-  } else if (isIndexNamesFormElement(indicesFormElement)) {
-    return (
-      <>
-        <IndexNamesConfigurationPanel
-          isLoading={isLoading}
-          isReadOnly={isReadOnly}
-          indexNamesFormElement={indicesFormElement}
-          onSwitchToIndexPatternReference={switchToIndexPatternReference}
-        />
-      </>
-    );
-  } else {
-    return null;
-  }
+          trackSwitchToIndexPatternReference({
+            metric: 'configuration_switch_to_index_pattern_reference',
+          });
+        }}
+        disabled={isReadOnly}
+      >
+        {isIndexPatternFormElement(indicesFormElement) && (
+          <IndexPatternConfigurationPanel
+            isLoading={isLoading}
+            isReadOnly={isReadOnly}
+            indexPatternFormElement={indicesFormElement}
+          />
+        )}
+      </EuiCheckableCard>
+      <EuiSpacer size="m" />
+
+      <EuiCheckableCard
+        id="indexNames"
+        label={
+          <EuiTitle size="xs">
+            <h2>
+              <FormattedMessage
+                id="xpack.infra.sourceConfiguration.indicesSectionTitle"
+                defaultMessage="Indices"
+              />
+            </h2>
+          </EuiTitle>
+        }
+        name="indexNames"
+        value="indexNames"
+        checked={isIndexNamesFormElement(indicesFormElement)}
+        onChange={() => {
+          if (indicesFormElement.initialValue?.type === 'index_name') {
+            indicesFormElement.updateValue(() => indicesFormElement.initialValue);
+          } else {
+            indicesFormElement.updateValue(() => ({
+              type: 'index_name',
+              indexName: '',
```

Comment:
Would it make sense to pre-populate the default value here?

Reply:
My feeling is no. I think we want it to be an intentional decision, below we have the helper text for the recommended value if the user ever feels lost.